concept/type/plays.feature

```diff
@@ -0,0 +1,2123 @@
+    When <root-type>(<type-name>) set plays: parentship:parent
+    When <root-type>(<type-name>) get plays(parentship:parent) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotation contain: @<annotation>
+    Then <root-type>(<type-name>) set plays: contract:contractor
+    Then <root-type>(<type-name>) get plays(contract:contractor) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(contract:contractor) get annotation contain: @<annotation>
+    When transaction commits
+    When connection open schema transaction for database: typedb
+    Then <root-type>(<type-name>) get plays(contract:contractor) get annotation contain: @<annotation>
+    Then <root-type>(<type-name>) set plays: parentship:parent
+    Then <root-type>(<type-name>) get plays(parentship:parent) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotation contain: @<annotation>
+    Examples:
+      | root-type | type-name   | annotation |
+      | entity    | person      | card(1, 2) |
+      | relation  | description | card(1, 2) |
+
+  Scenario Outline: <root-type> types can redeclare plays as plays with @<annotation>
+    When put relation type: contract
+    When relation(contract) create role: contractor
+    When put relation type: parentship
+    When relation(parentship) create role: parent
+    When put relation type: marriage
+    When relation(marriage) create role: spouse
+    When <root-type>(<type-name>) set plays: contract:contractor
+    When <root-type>(<type-name>) set plays: parentship:parent
+    When <root-type>(<type-name>) set plays: marriage:spouse
+    Then <root-type>(<type-name>) set plays: contract:contractor
+    Then <root-type>(<type-name>) get plays(contract:contractor) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(contract:contractor) get annotations contain: @<annotation>
+    When transaction commits
+    When connection open schema transaction for database: typedb
+    Then <root-type>(<type-name>) set plays: parentship:parent
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotations is empty
+    When <root-type>(<type-name>) get plays(parentship:parent) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotations contain: @<annotation>
+    Then <root-type>(<type-name>) get plays(marriage:spouse) get annotations is empty
+    When <root-type>(<type-name>) get plays(marriage:spouse) set annotation: @<annotation>
+    Then <root-type>(<type-name>) get plays(marriage:spouse) get annotations contain: @<annotation>
+    Examples:
+      | root-type | type-name   | annotation |
+      | entity    | person      | card(1, 2) |
+      | relation  | description | card(1, 2) |
+
+    # TODO: We set annotations independently now. Is the Scenario still relevant? I think so.
```

Comment:
Same as for `owns`.

Reply:
I think this is ok, we have to decide a language + concept API behaviour for annotation updating now